src/contrib/google/sheets/bricks/append.test.ts

```diff
@@ -73,3 +74,29 @@ describe("Normalize shape", () => {
     ]);
   });
 });
+
+describe("checkForBlankIntermediateColumns", () => {
+  it("passes for empty headers", () => {
+    expect(() => {
+      checkForBlankIntermediateColumns([]);
+    }).not.toThrow();
+  });
+
+  it("passes for valid headers", () => {
+    expect(() => {
+      checkForBlankIntermediateColumns(["foo", "bar"]);
+    }).not.toThrow();
+  });
```

Comment:
NIT: I think this is a better test?

```suggestion
  it("passes for valid headers", () => {
    expect(() => {
      checkForBlankIntermediateColumns(["foo", "bar", ""]);
    }).not.toThrow();
  });
```

Reply:
This situation is not possible, the api does not return empty cells at the end of a row. Otherwise it's not really clear where it would "stop," it could just return 999 empty cells or whatever.